src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -75,23 +85,6 @@ public String toString() {
     return referenceName + ':' + start + ':' + end;
   }
 
-  public List<Contig> getShards(long numberOfBasesPerShard) {
-    double shardCount = Math.ceil(end - start) / (double) numberOfBasesPerShard;
-    List<Contig> shards = Lists.newArrayList();
-    for (int i = 0; i < shardCount; i++) {
-      long shardStart = start + (i * numberOfBasesPerShard);
-      long shardEnd = Math.min(end, shardStart + numberOfBasesPerShard);
-
-      shards.add(new Contig(referenceName, shardStart, shardEnd));
-    }
-    Collections.shuffle(shards); // Shuffle shards for better backend performance
-    return shards;
-  }
-
-  public List<Contig> getShards() {
-    return getShards(DEFAULT_NUMBER_OF_BASES_PER_SHARD);
-  }
-
   public SearchVariantsRequest getVariantsRequest(String variantSetId) {
```

Comment:
In my opinion this method belongs in some variant-related class, not here. It would be something like an interface that's implemented by a VCF file reader and a Genomics API client.

The benefit is that (i) others can add other implementations if they want (reading from GCS, Hadoop, whatever comes next), and (ii) the contig class changes less (this is good because lots of code will come to depend on contig).